src/devices/Pn532/Pn532.cs

```diff
@@ -0,0 +1,1623 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Pn532.ListPassive;
+using Iot.Device.Pn532.RfConfiguration;
+using Iot.Device.Rfid;
+using IoT.Device.Pn532;
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.IO;
+using System.IO.Ports;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// PN532 RFID/NFC reader
+    /// </summary>
+    public class Pn532 : RfidWriteRead, IDisposable
+    {
+        // Timeout 
+        private const int ReadTimeOut = 500;
```

Comment:
is this based on spec? if not please make them public properties or move them to something SerialPort specific (haven't read the rest of the code to see how this is used)

Reply:
It is actually depending of the communication protocole used. Those 500 ms are higher values than in all modes (except 19.2K and 9.6K HSU where it can be higher). So 'Ill make it public. With this default value.